doc/api/crypto.md

```diff
@@ -5453,6 +5456,15 @@ comparing HMAC digests or secret values like authentication cookies or
 must have the same byte length. An error is thrown if `a` and `b` have
 different byte lengths.
 
+This function does not compare the elements of `a` and `b` directly. Instead, it
```

Comment:
What I meant before was adding the note about "underlying bytes" instead of adding this text. If we state we're comparing the underlying bytes, then this text isn't very useful and makes things more confusing.

Reply:
I agree. I'd just leave it at "compares bytes" and give it one or two examples. 0 vs. -0 is probably the easiest to understand but different NaNs could work too, e.g.:
```js
const x = new BigInt64Array(["0x7ff0000000000001", "0xfff0000000000001"])
const a = new Float64Array(x.buffer)
const b = new Float64Array([NaN, NaN])
crypto.timingSafeEqual(a, b) // false
```

Reply:
> On a related note, you could use `Object.is()` instead, which should give you the equality checking you're looking for.

Then I guess this earlier comment wasn't correct? `Object.is()` is true, unlike `timingSafeEqual`.

Reply:
> Then I guess this earlier comment wasn't correct? Object.is() is true, unlike timingSafeEqual.

As I said:

> I was referring to the equality checks inside the .every() callbacks.

To clarify further:

```js
Object.is(0, -0) === false // so of course `timingSafeEqual()` returns `false`
Object.is(NaN, NaN) === true // so of course `timingSafeEqual()` returns `true`
```

Reply:
@mscdex Your comment appears to imply some (obvious) connection between `Object.is()` and `timingSafeEqual()`, but @bnoordhuis' example seems to contradict that:

```js
const a = new Float64Array(new BigInt64Array([0x7ff0000000000001n]).buffer);
const b = new Float64Array(new BigInt64Array([0xfff0000000000001n]).buffer);
Object.is(a[0], b[0]); // true
crypto.timingSafeEqual(a, b); // false
```